### PR TITLE
feat: TextStyle Props Add in Table and Cell

### PR DESCRIPTION
**Pull request by pushpender-singh-ap**
### Summary

```
DataTable.Title add new props for text style (prop name: textStyle)
DataTable.Cell add new props for text style (prop name: textStyle)

Now users can add change color easily if they need.
```

<-- What existing problem does the pull request solve? Can you solve the issue with a different approach? -->
```
I was facing a problem in Table and cell I don't find any color props for color change so that is why I added some changes and make a pr for this.
```

### CODE REVIEW

**Comment by pushpender-singh-ap** [comment]
@lukewalczak please review this and make the alpha version so we can use this asap.

**Comment by lukewalczak** [comment]
Hey @pushpender-singh-ap I'm currently very busy with adjusting components to Material You, so I cannot guarantee I will review this PR soon :( In the meantime you can install paper library from your fork or use patch-package.

**Comment by pushpender-singh-ap** [comment]
@lukewalczak no issue when you free please review in meantime i am using my  fork library.

File: src/components/DataTable/DataTableTitle.tsx [hunk 1]
```diff
@@ -64,7 +66,7 @@ type Props = React.ComponentPropsWithRef<typeof TouchableWithoutFeedback> & {
  *           >
  *             Dessert
  *           </DataTable.Title>
- *           <DataTable.Title numeric>Calories</DataTable.Title>
+ *           <DataTable.Title colorText="#FFF" numeric>Calories</DataTable.Title>
```
Comment (lukewalczak) [inline src/components/DataTable/DataTableTitle.tsx:69]: I guess it should be removed

File: src/components/DataTable/DataTableCell.tsx [hunk 2]
```diff
@@ -18,6 +18,7 @@ type Props = $RemoveChildren<typeof TouchableRipple> & {
    */
   onPress?: () => void;
   style?: StyleProp<ViewStyle>;
+  textStyle?: StyleProp<TextStyle>;
```
Comment (lukewalczak) [inline src/components/DataTable/DataTableCell.tsx:21]: Please add prop description.

File: src/components/DataTable/DataTableTitle.tsx [hunk 3]
```diff
@@ -35,6 +36,7 @@ type Props = React.ComponentPropsWithRef<typeof TouchableWithoutFeedback> & {
    */
   onPress?: () => void;
   style?: StyleProp<ViewStyle>;
+  textStyle?: StyleProp<TextStyle>;
```
Comment (lukewalczak) [inline src/components/DataTable/DataTableTitle.tsx:39]: Please add prop description.

Comment (pushpender-singh-ap) [reply src/components/DataTable/DataTableTitle.tsx:42]: Description added if you thing any change please inform me

File: src/components/DataTable/DataTableCell.tsx (hunk 2 above)
Comment (pushpender-singh-ap) [reply src/components/DataTable/DataTableCell.tsx:24]: Description added if you thing any change please inform me

File: src/components/DataTable/DataTableTitle.tsx (hunk 1 above)
Comment (pushpender-singh-ap) [reply src/components/DataTable/DataTableTitle.tsx]: Yes, removed now

File: src/components/DataTable/DataTableCell.tsx [hunk 4]
```diff
@@ -18,6 +18,10 @@ type Props = $RemoveChildren<typeof TouchableRipple> & {
    */
   onPress?: () => void;
   style?: StyleProp<ViewStyle>;
+  /**
+   * text Content style of the `DataTableCell`.
```
Comment (lukewalczak) [inline src/components/DataTable/DataTableCell.tsx:22]:
```suggestion
   * text Content style of the `DataTableCell`.
```
```suggestion
   * Text content style of the `DataTableCell`.
```

Comment (lukewalczak) [inline src/components/DataTable/DataTableCell.tsx:22]:
```suggestion
   * text Content style of the `DataTableCell`.
```
```suggestion
   * Text content style of the `DataTableCell`.
```

Comment (pushpender-singh-ap) [reply src/components/DataTable/DataTableCell.tsx]: DONE

**Comment by pushpender-singh-ap** [comment]
@lukewalczak any change required?

**Pull request merged**